src/components/NoteDetails/NoteDetailsContainer.js

```diff
@@ -16,8 +19,11 @@ const mapStateToProps = ({
 };
 
 const mapDispatchToProps = () => ({
-  onNoteNameChange: noteName => {
-    action(EDITOR_NOTE_NAME_CHANGED_ACTION, noteName);
+  onNoteNameChange: ({ note, newNoteName }) => {
+    action(EDITOR_NOTE_NAME_CHANGED_ACTION, { note, newNoteName });
+  },
+  onNoteContentChange: ({ note, noteText }) => {
+    action(EDITOR_NOTE_CONTENT_CHANGED_ACTION, { note, noteText });
```

Comment:
the function name is `onNoteContentChange`, but it accepts `noteText`, not `noteContent`. Please use one name to avoid confusion.